sklearn/utils/random.py

```diff
@@ -6,9 +6,9 @@
 import array
 
 from sklearn.utils import check_random_state
-from ._random import sample_without_replacement
+from ._random import _cast_py, _our_rand_r_py, sample_without_replacement
 
-__all__ = ['sample_without_replacement']
+__all__ = ['sample_without_replacement', '_our_rand_r_py', '_cast_py']
```

Comment:
I am not sure we want these functions in `__all__`, since they are private and only used for testing.